WordPressKit/Sources/WordPressKit/Models/Stats/Time Interval/StatsFileDownloadsTimeIntervalData.swift

```diff
@@ -49,11 +55,21 @@ extension StatsFileDownloadsTimeIntervalData: StatsTimeIntervalData {
         }
 
         let fileDownloads: [StatsFileDownload] = fileDownloadsDict.compactMap {
-            guard let file = $0["filename"] as? String, let downloads = $0["downloads"] as? Int else {
+            guard let file = $0["filename"] as? String,
+                  let downloads = $0["downloads"] as? Int,
+                  let relativeURLString = $0["relative_url"] as? String,
+                  let downloadURLString = $0["download_url"] as? String,
+                  let downloadURL = URL(string: downloadURLString)
```

Comment:
Just to be safe, let's make `downloadURL` optional, so it's ever not present in production, the app will still display what it can.